stryker-mutator/stryker
# feat(karma-runner): implement v2 test runner api

* Implement new v2 test runner API (see #2249 )
* Make sure to not report coverage when it isn't needed
* Add a framework adapter file that adds mutant coverage to the karma result, to be picked up by the stryker reporter
* Add test filtering capability when running with jasmine or mocha
* Add integration tests on instrumented code.
* Some more housekeeping.
